content/resources/user/submit-data.md

```diff
@@ -283,3 +284,8 @@ For example, the following link would download a single TSV containing all the `
 https://gen3.datacommons.io/api/v0/submission/example/training/export/?node_label=core_metadata_collection&format=tsv
 
 The links in the downloaded TSV can be updated by filling in the submitter_ids of the corresponding parent records, saving, and re-submitting the file to the data portal website using 'Upload File' as done in [step 4](#more-about-specifying-required-links).
+
+
+
+## 6. Linking Data from external Data Clouds to Gen3 Data Commons
+It is possible to link data on Gen3 that is stored on other cloud services (Amazon Web Services AWS, Google Cloud Storage GCS) by a process called DIIRM (*Data Ingestion, Integration, and Release Management*). If you have a bucket of files and want to link the data to Gen3 you can find [here a step-by-step guide for this purpose](https://gen3.org/resources/user/submit-data/sower).
```

Comment:
I would leave out the part about "DIIRM". That's an internal project name that I don't think users will need to know or care about.

Reply:
deleted this whole section and made edits according to comment above.